internal/cli/kraft/run/run.go

```diff
@@ -252,6 +256,11 @@ func (opts *RunOptions) Run(ctx context.Context, args []string) error {
 		return err
 	}
 
+	err = opts.discoverNetworkController(ctx)
+	if err != nil {
```

Comment:
You can inline this:
```go
if err := opts.discoverNetworkController(ctx); err != nil {
```

Reply:
Thanks! I inlined this one and the one above for consistency